subsys/net/lib/mcumgr_smp_client/include/mcumgr_smp_client.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file mcumgr_smp_client.h
+ *
+ * @defgroup mcumgr_smp_client for Download library and update that over DFU SMP target
```

Comment:
```suggestion
 * @defgroup mcumgr_smp_client A client for downloading a firmware update over SMP
```
Something like that, but you would have to make it more precise